docs/section11/Rule_11-9.md

```diff
@@ -0,0 +1,38 @@
+# Service_Water_Heating - Rule 11-9
+**Schema Version:** 0.0.38  
+
+**Mandatory Rule:** FALSE
+
+**Rule ID:** 11-9
+
+**Rule Description:** "The baseline system must be sized according to Standard 90.1 2019, Section 7.4.1."  
```

Comment:
Very small thing but suggest removing the quotes since they are not in other RDSs.